Extension/walkthrough/install-gcc-linux.md

```diff
@@ -0,0 +1,20 @@
+
+# Install a C++ compiler on Linux
+If you're doing C++ development for Linux, we recommend installing the GCC compiler. Installing GCC is simple, just follow these three steps:
+
+1. Run the following command from the terminal window to update the Ubuntu package lists. An out-of-date Linux distribution can sometimes interfere with attempts to install new packages.
+
+    ```bash
+    sudo apt-get update
+    ```
+
+2. Install the GNU compiler tools and the GDB debugger with this command:
+
+    ```bash
+    sudo apt-get install build-essential gdb
+
+3. Verify GCC is installed by running the following command. You should see a copyright message with the version and basic usage description.
```

Comment:
I don't see a "basic usage description":
```
~$ gcc --version
gcc (Ubuntu 10.3.0-1ubuntu1~20.04~2) 10.3.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Reply:
Updated to "You should see a copyright message and information about the version of GCC you're using."